includes/CLI/Plugin_Check_Command.php

```diff
@@ -62,6 +62,10 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * [--checks=<checks>]
 	 * : Only runs checks provided as an argument in comma-separated values, e.g. i18n_usage, late_escaping. Otherwise runs all checks.
 	 *
+	 * [--exclude-checks=<checks>]
```

Comment:
You call this `ignore-checks` everywhere else in this PR. I don't feel strongly about either way, but we should go with a consistent term throughout.

I personally prefer `exclude-checks` I think, but either way WFM.

Reply:
Makes sense, let's settle on _exclude_